cartoframes/data/observatory/category.py

```diff
@@ -1,31 +1,21 @@
 from __future__ import absolute_import
 
-from .entity import SingleEntity, EntitiesList
+from .entity import CatalogEntity
 from .repository.category_repo import get_category_repo
 from .repository.dataset_repo import get_dataset_repo
 
 
 _CATEGORY_ID_FIELD = 'id'
 
 
-class Category(SingleEntity):
+class Category(CatalogEntity):
 
     id_field = _CATEGORY_ID_FIELD
     entity_repo = get_category_repo()
 
-    @classmethod
-    def _get_entities_list_class(cls):
-        return Categories
-
     def datasets(self):
```

Comment:
It shold be a property

Reply:
Ups, I missed those. Fixed for all methods.